integration/localnet/bootstrap.go

```diff
@@ -686,18 +686,13 @@ func prepareObserverService(i int, observerName string, agPublicKey string, prof
 	observerService := Service{
```

Comment:
Is it possible to make observer less of a special case by removing snowflake functions and switching to their generic counterparts?
* `prepareObserverService` -> `prepareService`
* `prepareObserverProfilerFolder` -> `prepareCommonHostFolders`
* `prepareObserverDataFolder` -> `prepareCommonHostFolders`
* `getObserverProfilerDir` -> nil
*  etc

Reply:
Addressed: https://github.com/GetElastech/flow-go/commit/f094c4a126a30eae6004bfc7a6d903b058810de2